### ISSUE TITLE
Custom argument type for *args.

### ISSUE THREAD

**Issue by JohnnySG** [cunnane/xloil#41]
I have the following function:
```
@xlo.converter(range=True)
def arg_array(x):
    if isinstance(x, xlo.Range):
        x = np.array(x.value, ndmin=2)
        x[x == None] = 0
    else:
        x = np.array(x, ndmin=2)
    return x


@xlo.func
def vstack(a: arg_array, b: arg_array):
    return np.vstack((a, b))
```
It worked well, however I want to get variable arguments, as below:
```
@xlo.func
def vstack(*args: arg_array):
    return np.vstack(args)
```
This time, the type _arg_array_ seems not work!

**Comment by cunnane** [comment]
This is fixed in v0.17.9

**Comment by JohnnySG** [comment]
Thank you for your work!
